news.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "link": "https://www.standardmedia.co.ke/business/article/2001253732/businesses-rate-august-worst-month-in-years", 
+    "img": "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png", 
+    "title": "Businesses rate August \u2018worst\u2019 month in years"
+  }, 
+  {
+    "link": "https://www.standardmedia.co.ke/business/article/2001253765/the-amount-of-money-judiciary-makes-from-election-petitions", 
+    "img": "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png", 
+    "title": "The amount of money Judiciary makes from election petitions"
+  }, 
+  {
+    "link": "https://www.standardmedia.co.ke/business/article/2001253585/system-upgrade-to-take-m-pesa-offline", 
+    "img": "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png", 
+    "title": "System upgrade to take M-Pesa offline"
+  }, 
+  {
+    "link": "https://www.standardmedia.co.ke/business/article/2001253652/kcb-s-lions-den-show-back-on-ktn-channel", 
+    "img": "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png", 
+    "title": "KCB\u2019s Lions den show back on KTN channel"
+  }
+]
```

Comment:
Is this file a test or the actual scrape?

Reply:
It's a test the actual one is the one above.

Reply:
If a test file, should go to `test` folder.